mavros_extras/src/plugins/fake_gps.cpp

```diff
@@ -266,6 +225,9 @@ class FakeGPSPlugin : public plugin::PluginBase,
 		old_ecef = current_ecef;
 
 		UAS_FCU(m_uas)->send_message_ignore_drop(fix);
+
+		// Throttle incoming messages to 5hz
+		gps_rate.sleep();
```

Comment:
Done. Applied the previous method with conversion from `Rate` to `Duration` so then I'm sure it will function properly.